packages/realm-network-transport/types/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Timer related stuff
```

Comment:
We really shouldn't be declaring this stuff ourselves.

Reply:
I think we do.
I'm failing to find a good dependency declaring types of the intersection of the Node.js and browser / DOM runtime.
I'll be happy to change this if you can find an alternative that doesn't define globals that are not available in the other.

Reply:
Nope.
They are [included in the compilation](https://github.com/realm/realm-js/blob/kh/realm-web-iife/packages/realm-network-transport/tsconfig.json#L22) and as long as that file is never imported into the dependency tree, they will not end up in the bundle.

Reply:
I'm still not convinced that this is a good idea. However, I also don't want to block our progress. How about we move forward with this for now, but after everyone is back from vacation, we discuss as a full team and decide how we feel about this? Does that work for you?